spec/mcode.spec.ts

```diff
@@ -647,7 +649,7 @@ describe('checkHistologyMorphologyFilterLogic-NonInflammatoryInvasive', () => {
   });
 });
 
-/* Stage Logic Tests */
+/* Stage Logic Tests */ //- needs updated
```

Comment:
Does this still need updating or has it been updated? I'm confused about this comment.

Reply:
Sorry, I definitely meant to delete those before I made the PR. Both comments have been removed.